src/index.css

```diff
@@ -7,6 +7,10 @@
 @tailwind components;
 @tailwind utilities;
 
+.scrollbar-stable {
+  scrollbar-gutter: stable;
```

Comment:
Modern CSS I don't even know. Needed to look this up: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter  :D  
In the old days this was some overflow hack I think.